src/components/accordion/accordion-header.tsx

```diff
@@ -44,7 +44,7 @@ const AccordionButton = styled(ReachAccordion.AccordionButton)`
 const AccordionChevron = (props: AccordionChevronProps) => {
   return (
     <div className={props.className}>
-      <ChevronSVG />
+      <Icon show="chevron_up" />
     </div>
   );
 };
```

Comment:
Have a look at the icon component. It does exactly the same as AccordionChevron. So you can delete the `AccordionChevron` and apply the style directly to the icon component (i.e. `const StyledIcon = styled(Icon)<{open: boolean}>` instead of `const AccordionChevronStyled = styled(AccordionChevron)`). If you do it this way you have to remember to add `show="chevron_up"` to the new `StyledIcon` in the `AccordionHeader`.

Reply:
Thank you for that suggestion, looks way cleaner.
I changed it 👍